requirements-dev.txt

```diff
@@ -1,5 +1,6 @@
 numpy
 protobuf
+opencv-python
```

Comment:
It should not be needed since it is also added at the last line.

Reply:
True. Fixed